.circleci/config.yml

```diff
@@ -502,18 +541,18 @@ jobs:
         type: string
         default: "chrome"
     docker:
-      - image: cimg/node:16.16.0-browsers
+      - image: $AWS_ACCOUNT_ID_DEV.dkr.ecr.$AWS_REGION.amazonaws.com/navigator_builder:node_v16_16
+        aws_auth:
+          aws_access_key_id: $AWS_ACCESS_KEY_ID_DEV
+          aws_secret_access_key: $AWS_SECRET_ACCESS_KEY_DEV
     steps:
       - set-cypress-env-vars
-      - attach_workspace:
-          at: .
+      - persist_to_workspace:
```

Comment:
Why was this changed from attach_workspace to persist_to_workspace? 
As this is the integration test, you want to be using the workspace that has been saved previously. There wouldn't be anything to persist at this point

Reply:
Reverted it back to attach_workspace.